app/scripts/models/auth_brokers/oauth.js

```diff
@@ -119,6 +119,8 @@ define([
     },
 
     afterSignIn: function (account, additionalResultData) {
+      // Signal to the RP that this was an existing account sign-in.
+      additionalResultData.action = 'signin';
```

Comment:
These magic strings should probably live in [lib/constants.js](https://github.com/mozilla/fxa-content-server/blob/4898dc006863fe26746b7562cc5dad78c36ca2f5/app/scripts/lib/constants.js)